forta-emitting-upgraded/src/agent.js

```diff
@@ -43,9 +46,12 @@ function provideHandleTransaction(getContracts) {
           alertId: "FORTA-EMIT-UPGRADED",
           severity: FindingSeverity.Low,
           type: FindingType.Info,
+          protocol: "forta",
           metadata: {
             name: contact.name,
             address: contact.address,
+            mintedBy: fromLowerCase,
```

Comment:
The feedback was "Lets also put chainId and implementation address in the metadata". I see the implementation field but not the chainId. Also I don't think we need the mintedBy field